packages/pangea-sdk/pangea/deep_verify.py

```diff
@@ -204,21 +203,24 @@ def deep_verify(audit: Audit, file: io.TextIOWrapper) -> Errors:
             else:
                 hot_path = path[:-cold_path_size]
                 cold_path = path[-cold_path_size:]
-            
+
             cold_idx = path2index(tree_size, cold_path)
             if cold_idx != leaf_index:
                 errors["wrong_buffer"] += 1
 
             hot_idx = path2index(len(events_by_idx), hot_path)
-            if hot_idx in hot_out_of_order:
-                hot_out_of_order.remove(hot_idx)
-            elif hot_idx != i:
-                hot_out_of_order.add(hot_idx)
-
-        if hot_out_of_order:
-            errors["missing"] += len(hot_out_of_order)
+            hot_indexes.add(hot_idx)
+
+        hot_indexes_diff = set(range(len(events_by_idx))) - hot_indexes
+        if len(hot_indexes_diff) > 0:
+            errors["missing"] += len(hot_indexes_diff)
+            print(f"missing hot indexes: {hot_indexes_diff}")
```

Comment:
I think this is too much information and too internal (we don't speak about hot/cold with the users)